dataset-spec/README.md

```diff
@@ -0,0 +1,85 @@
+# Dataset Spec for STAC
+
+## Introduction
+
+One topic of interest has been the search of datasets*, instead of within a dataset, i.e. in (sub-)catalogs, items and assets. [STAC](https://github.com/radiantearth/stac-spec) is focused on search within a dataset, but it includes some simple constructs to catalog datasets. This could be an independent spec that STAC uses, and others can also independently use, to describe datasets in a lightweight way.
+
+*\* There is no standardized name for the concept we are describing here. Others called it: dataset series (ISO 19115), collection (CNES, NASA), dataset (JAXA), dataset series (ESA), product (JAXA).*
+
+## Core
+
```

Comment:
In the EO spec we have gsd (Ground Sample Distance) at the top level, and it's also provided per band because resolution may vary by band.  At the top level it represents the best resolution to enable searching. I think it makes sense at the Dataset level rather than the Item level.

Reply:
I'm not sure I entirely get what you are saying.  
I think datasets are a concept which will be used in general with STAC - although I'm still not sure if the intention is that datasets are always present and if they are themselves part of core. I personally think they should part of core, the include core fields: temporal and spatial extent are the unions of core fields, license, provider...

So the EO extension, or any extension, should define additions to both the Dataset and the Item.

Reply:
I hope they are part of the core and always present, but not sure what others think.

I agree with what you are saying about extensions and that answers basically my question. So I would expect that the additional fields we are proposing for datasets will be incorporated into the EO extension. If you are okay with that, I'd already move them to the eo extension in the branch we are currently working on. They are currently a bit badly located in the dataset-spec.

Reply:
Yes, I think those additional fields belong in the EO extension...although I think that I'd still like to see some of the non-varying asset info in Datasets, but I'll make the case and provide some examples in the EO extension.

Reply:
Which of these non-varying info would that be?

Reply:
Great, then I'd like to have your support here (regarding global extensions): #186 ;-)

Sure, I think more issues would make sense! I'd like a proposal on this. Would you mind doing that? I don't think it's simply copy and paste (minus url) from the assets sepc in the items?